### PR TITLE
update all the things

### PR DESCRIPTION
**Pull request by e0d**
@sarina if we are going to release the notifier, I'd like to also do standard upgrade hygiene.

Opened as me.

### CODE REVIEW

**Comment by sarina** [comment]
Hmm. I know nothing about the notifier; I was only updating the translations. Presumably, we have to figure out how to get all these tests to pass. I believe the T&L team is the owner here, but I'm doubtful they have the time to figure this out.

**Comment by e0d** [comment]
I think the minimal requirement for releasing is upgrading requests and django which have known exploits, if TNL owns this I'll reach out to them.

**Comment by e0d** [comment]
@jimabramson can you speak to confidence that passing tests means we are AOK to deploy?

File: requirements.txt [hunk 1]
```diff
@@ -1,26 +1,26 @@
-Django==1.4.5
-amqp==1.0.12
-APScheduler==2.1.1
-autopep8==0.9.2
-billiard==2.7.3.28
-celery==3.0.19
-coverage==3.6
-django-celery==3.0.17
-django-configurations==0.3
+Django==1.4.13
```
Comment (sarina) [inline requirements.txt]: Last django is 1.4.22

**Comment by e0d** [comment]
@robrap @cahrens I believe that you own this project.  Sarina would like to release it update translations.  If we are releasing, I'd also like to update all the libraries it uses.  Does anyone from  your team have context on this application?

**Comment by cahrens** [comment]
No, I'm afraid not. We can choose someone to tag along and learn though!

I think the only person with notifier knowledge is @jimabramson.

**Comment by e0d** [comment]
Let me know whom you tag.

**Comment by jimabramson** [comment]
@e0d It's *probably* fine, but of course I can't guarantee that none of these updates will change behavior.  I doubt we'd balk at changing any one of them, and I agree with getting things up to date, but identifying the cause of a problem after this many updates might be harder than if done piecemeal.  

Deploying this to a sandbox and giving it a test digest run would provide more certainty.

A few key libraries had major version increments, e.g. Kombu, Billiard, APScheduler.  If they have not already been, changelogs should be looked over to see if any warnings or compatibility issues surface.

**Comment by sarina** [comment]
Can we absolutely not deploy translations until this is resolved? I don't
know who is going to be able to do all this testing - although I do agree
it's necessary.

**Comment by cahrens** [comment]
@e0d -- @robrap can tag along and learn

**Comment by jimabramson** [comment]
@robrap @e0d i'm happy to answer any questions / provide context as you work through this

**Comment by e0d** [comment]
@sarina if there's urgency in getting the translations out, we can do them without the other changes.  If we can wait a week, I'd happily do that.  Defer to you.

**Comment by sarina** [comment]
Waiting a week sounds good; an open ended wait is not.

**Comment by robrap** [comment]
@e0d It sounds like we can run the following test on Stage once this has been deployed there:
1. Adjust scheduler in stage to 10 minute interval.
2. Someone can follow the instructions in your page to retrieve digests and make changes that would require a digest to be sent. (I'm happy to do this.)
3. Verify the following: 
- make sure you retrieve an email (i.e. it's finding users and updates for digests)
- make sure the text output looks good
- make sure the html output looks good

Just tag me when you want me to test.

**Comment by robrap** [comment]
By "your page", I meant: https://openedx.atlassian.net/wiki/x/OYVUAQ

Hopefully we won't need this, but if we need to troubleshoot, we can use any of the following params from the forums_digest call you have documented: https://github.com/edx/notifier/blob/master/notifier/management/commands/forums_digest.py#L40-L77

**Comment by e0d** [comment]
@robrap @sarina any objections to merging and cutting an RC from master?

**Comment by e0d** [comment]
@fredsmith FYI

**Comment by robrap** [comment]
I have no objection, but I also don't understand the implication.  Are you asking whether we should do the testing of this on stage with the RC?  Again, no objection, as long as it is simple to revert it out should we find an issue.  Hopefully it all goes smoothly.

**Comment by e0d** [comment]
Signify your lack of objection with a thumb :)

**Comment by robrap** [comment]
:+1: Assuming your request for a thumb is confirming my understanding of the plan.

**Pull request merged**